Gemfile

```diff
@@ -1,8 +1,10 @@
 source 'https://rubygems.org'
 
-ruby '2.3.1'
+ruby '2.3.3'
```

Comment:
this is basically a total rewrite using the existing repo name. Bumping ruby versions didn't feel like a big deal to me for code that has never run/deployed anywhere.